packages/std-contracts/src/test/SampleSystem.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+import "../systems/StdSystem.sol";
+import { IWorld } from "solecs/interfaces/IWorld.sol";
+import { LibStorage as s } from "../libraries/LibStorage.sol";
+import { console } from "forge-std/console.sol";
+
+uint256 constant ID = uint256(keccak256("system.sample"));
+
+contract SampleSystem is StdSystem {
+  constructor(IWorld _world, address _components) StdSystem(_world, _components) {}
+
+  function execute(bytes memory arguments) public returns (bytes memory) {
+    if (address(s.world()) == address(world)) {
+      console.log("World is correct");
+    }
+    if (address(s.comp()) == address(components)) {
+      console.log("Components are correct");
+    }
```

Comment:
I think we should change this to use `require` (or `revert`) so the execute function reverts if the world is not correct and the test fails

Reply:
Changed to revert in [16e2033](https://github.com/latticexyz/mud/pull/264/commits/16e203374934ad3df1115076c0083939d69a1060)